test/unit/eval_algorithms/test_common.py

```diff
@@ -90,11 +90,17 @@ def test_save_dataset(tmp_path, file_name):
 
             if json_obj[DatasetColumns.MODEL_INPUT.value.name] == "hello":
                 assert json_obj[DatasetColumns.CATEGORY.value.name] == "Age"
-                assert json_obj["scores"] == [{"name": "rouge", "value": 0.5}, {"name": "bert_score", "value": 0.42}]
+                assert json_obj["scores"] == [
+                    {"name": "rouge", "value": 0.5, "error": None},
+                    {"name": "bert_score", "value": 0.42, "error": None},
```

Comment:
Can you update the `save_dataset` such that the `error`field is not even written when it is `None`?

Reply:
Fixed in https://github.com/aws/fmeval/pull/297